AlfrescoApp/View Controllers/Accounts View Controllers/New Account/NewAccountViewController.m

```diff
@@ -441,7 +441,7 @@ - (void)constructTableCellsForAlfrescoServer
         protocolCell.titleLabel.text = NSLocalizedString(@"accountdetails.fields.protocol", @"HTTPS protocol");
         self.protocolSwitch = protocolCell.valueSwitch;
         [self.protocolSwitch addTarget:self action:@selector(protocolChanged:) forControlEvents:UIControlEventValueChanged];
-        BOOL isHTTPSOn = self.formBackupAccount.protocol ? [self.formBackupAccount.protocol isEqualToString:kProtocolHTTPS] : NO;
+        BOOL isHTTPSOn = self.formBackupAccount.protocol ? [self.formBackupAccount.protocol isEqualToString:kProtocolHTTPS] : YES;
```

Comment:
Original code is like this:

BOOL isHTTPSOn = self.formBackupAccount.protocol ? [self.formBackupAccount.protocol isEqualToString:kProtocolHTTPS] : NO;
        if (self.activityType == AccountActivityTypeNewAccount)
        {
            isHTTPSOn = YES;
        }
        [self.protocolSwitch setOn:isHTTPSOn animated:YES];

I fact could just leave it at isHTTPSOn = YES;